toolchain/check/import_cpp.cpp

```diff
@@ -310,13 +310,28 @@ static auto MakeIntType(Context& context, IntId size_id) -> TypeExpr {
   return ExprAsType(context, Parse::NodeId::None, type_inst_id);
 }
 
-// Maps a C++ type to a Carbon type. Currently only 32-bit `int` is supported.
+// Maps a C++ type to a Carbon type. Currently only 32-bit `int` and 16-bit
+// `short` are supported.
```

Comment:
```suggestion
// Maps a C++ type to a Carbon type.
```
Rather than making this comment have to get modified with every change

Reply:
Don.e